tests/integration/opcache/disabled/test_computations.php

```diff
@@ -0,0 +1,78 @@
+<?php
+/*
+ * Copyright 2022 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/*DESCRIPTION
+Transaction event created and no errors despite creating spans
+for a HUGE number of calls.
+*/
+
+/*SKIPIF
+<?php
+
+require('skipif.inc');
+*/
+
+/*INI
+newrelic.distributed_tracing_enabled=1
+newrelic.transaction_tracer.threshold = 0
+error_reporting = E_ALL
+opcache.enable=0
+opcache.enable_cli=0
+opcache.file_update_protection=0
+opcache.jit_buffer_size=32M
+opcache.jit=function
+*/
+
+/*PHPMODULES
+zend_extension=opcache.so
```

Comment:
Does this still need to be in tests if the default is for opcache to be enabled?

Reply:
Good point, please see: [5061df7](https://github.com/newrelic/newrelic-php-agent/pull/752/commits/5061df70afadd56e54d08258142317996ae606af)